src/lanternfish/lanternfish.md

```diff
@@ -0,0 +1,45 @@
+# Lanternfish kata
+
+> Note: this is the day 6 of the Advent of Code 2021. The instructions are simplified to make them easier to understand. You can find the original instructions [here](https://adventofcode.com/2021/day/6).
+
+## The brief
+
+As you're swimming in the sea, you come across a massive school of lanternfish. You get curious about them: they must spawn quickly to reach such large numbers, but how quickly exactly?
```

Comment:
the reference to "exponential" in the original text is important, let's keep it

Reply:
I felt that might be too much help when tackling the 256d , but happy to keep